src/components/header-block/header-block.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import HeaderBlock from './header-block';
+
+export default {
+  component: HeaderBlock,
+  title: 'Components/HeaderBlock',
+} as ComponentMeta<typeof HeaderBlock>;
+
+const Template: ComponentStory<typeof HeaderBlock> = (args) => (
+  <HeaderBlock {...args}>
+    Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi
+    leo risus, porta ac consectetur ac, vestibulum at eros. Maecenas sed diam
+    eget risus varius blandit sit amet non magna.
+  </HeaderBlock>
+);
+
+export const Regular = Template.bind({});
+Regular.args = {
+  topline: 'Topline',
+  headline: 'Headline',
+  date: 'vor 3 Tagen',
+  author: 'von Fabian Dietenberger, Frontend Engineer, Satellytes',
+  illustration: 'rocket_011',
+  large: true,
```

Comment:
what's thay. do we have different sizes? If you look careful, the design demonstrates the mobile version too, so maybe you mixed that up ?
![Screen Shot 2021-11-19 at 14 21 31](https://user-images.githubusercontent.com/1701755/142629723-a2196989-c533-4b02-a2dc-8e50079cdf66.png)

Reply:
But there is for both, Mobile and Desktop, always a Regular and a Large state, isn't it?